data/json/items/book/misc.json

```diff
@@ -635,33 +654,21 @@
     "type": "BOOK",
     "name": "adventure novel",
     "description": "The stirring tale of a race against time, in search of a lost city located in the dark heart of the African continent.",
-    "weight": "404 g",
-    "volume": "750 ml",
+    "copy-from": "paperback_novel",
     "price": 850,
-    "price_postapoc": 1900,
-    "material": [ "paper" ],
-    "symbol": "?",
-    "color": "light_blue",
-    "intelligence": 4,
     "time": "20 m",
-    "chapters": 16,
-    "fun": 4
+    "fun": 3
   },
   {
     "id": "novel_buddy",
     "type": "BOOK",
     "name": "buddy novel",
     "description": "A gripping tale of two friends struggling to survive on the streets of New York City.",
+    "copy-from": "paperback_novel",
     "weight": "244 g",
     "volume": "500 ml",
     "price": 650,
-    "price_postapoc": 1700,
-    "material": [ "paper" ],
-    "symbol": "?",
-    "color": "light_blue",
-    "intelligence": 4,
     "time": "20 m",
-    "chapters": 16,
     "fun": 2
```

Comment:
You're getting an "assignment does not update value" here because this is the same fun value as the root item it's copied from.